planning/behavior_path_dynamic_avoidance_module/include/behavior_path_dynamic_avoidance_module/scene.hpp

```diff
@@ -253,16 +261,17 @@ class DynamicAvoidanceModule : public SceneModuleInterface
         std::remove_if(
           valid_object_uuids_.begin(), valid_object_uuids_.end(),
           [&](const auto & uuid) {
-            return counter_map_.count(uuid) == 0 || counter_map_.at(uuid) < max_count_;
+            return counter_map_.count(uuid) == 0 || counter_map_.at(uuid) < min_count_;
           }),
         valid_object_uuids_.end());
 
       // remove objects whose counter is lower than threshold
       const auto counter_map_keys = getAllKeys(counter_map_);
       for (const auto & key : counter_map_keys) {
-        if (counter_map_.at(key) < min_count_) {
+        if (counter_map_.at(key) == 0) {
```

Comment:
Can I ask the reason for this?

My intention is when the counter is as follows.
```
INVALID --- (min_count_) --- COUNTING --- (max_count_) --- VALID
```
VALID means that we decide to avoid, and INVALID is we do not have to count and ignore the object. So I use `min_count_` for the condition you changed.